ethereum-types/Cargo.toml

```diff
@@ -22,3 +22,4 @@ serde_json = "1.0.41"
 default = ["std", "serialize"]
 std = ["uint-crate/std", "fixed-hash/std", "ethbloom/std", "primitive-types/std"]
 serialize = ["std", "impl-serde", "primitive-types/serde", "ethbloom/serialize"]
+arbitrary = ["std", "serialize", "fixed-hash/arbitrary"]
```

Comment:
why is `serialize` required?

Reply:
would you mind adding `arbitrary` support for `uint` as well?

Reply:
Good point neither of `std` or `serialize` are actually required but were useful to me so I'll remove them. It's easy for the calling library to have `["default", "aribtrary"]`

Reply:
> why is `serialize` required?

Yep can do

Reply:
> would you mind adding `arbitrary` support for `uint` as well?

Yep can do